packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -1386,6 +1402,31 @@ class AccountInternal extends React.PureComponent {
     this.setState({ reconcileAmount: null });
   };
 
+  onCreateReconciliationTransaction = async diff => {
+    // Create a new reconciliation transaction
+    const reconciliationTransactions = realizeTempTransactions([
+      {
+        id: 'temp',
+        account: this.props.accountId,
+        cleared: true,
+        amount: diff,
+        date: formatDate(new Date(), 'yyyy-mm-dd'),
```

Comment:
Is there a nice util available that I could use here to generate the `current-date`? I'm not particularly loving the approach here. Open to suggestions.

Reply:
@MatissJanis Good point, it looks like there is, see `packages/loot-core/src/shared/months.js`:

```js
export function currentDay() {
  if (global.IS_TESTING) {
    return '2017-01-01';
  } else {
    return d.format(new Date(), 'yyyy-MM-dd');
  }
}

```

Reply:
Amazing! Thanks for the tip
https://github.com/actualbudget/actual/pull/260/commits/02c837c076fdd00f7faf99b349ce526fc1fa46ad